docs/user-guide/ignore-code.md

```diff
@@ -63,7 +63,8 @@ Stylelint supports complex, overlapping disabling & enabling patterns:
 ```
 
 > [!WARNING]
-> Comments within _selector and value lists_ are currently ignored.
+> Non-standard syntax comments (i.e. `//`) are only minimally supported.  
```

Comment:
```suggestion
> Configuration commands in non-standard syntax comments (e.g. `// stylelint-disable`) are only minimally supported.  
```

Reply:
To make it clear that we are not referring to support of comments, but rather the commands in such comments